### PR TITLE
feat(Provider): add "getMessageStatus" method

### PR DESCRIPTION
**Pull request by LuizAsFight**
this is a followup PR from #1284  as it was not providing a way for users to consume the new query

### CODE REVIEW

File: packages/providers/src/operations.graphql [hunk 1]
```diff
@@ -446,6 +446,12 @@ query getMessageProof(
   }
 }
 
+query getMessageStatus($nonce: Nonce!) {
+  messageStatus(nonce: $nonce) {
+    state
+  }
+}
+
```
Comment (arboleya) [inline packages/providers/src/operations.graphql:454]: Quick question — shouldn't this have been generated automatically by this script?

https://github.com/FuelLabs/fuels-ts/blob/10f54b873b05211a9240ba550a3ea33b639d471e/packages/providers/package.json#L21

**Review by arboleya — APPROVED** [review]
The good son returns home. 🙂

File: packages/providers/src/operations.graphql (hunk 1 above)
Comment (LuizAsFight) [reply packages/providers/src/operations.graphql:454]: no, the `build-operations` will build your `operations.graphql` file and create the `__generated__/operations.ts`file 

but all the queries in `operations.graphql` are written by us

Comment (arboleya) [reply packages/providers/src/operations.graphql:454]: Oh, right! Thank you.

**Pull request merged**